src/components/TileCatalogNew/TileCatalogNew.test.jsx

```diff
@@ -143,6 +143,51 @@ describe('TileCatalogNew', () => {
     expect(getByText('Tile 16')).toBeTruthy();
   });
 
+  it('TileCatalogNew pagination does not render with only one page', () => {
+    const { queryByText } = render(
+      <TileCatalogNew
+        title="Test Tile Catalog"
+        tiles={getTiles(4, 'Tile')}
+        numColumns={2}
+        numRows={2}
+      />
+    );
+
+    expect(queryByText('Next page')).toBeNull();
```

Comment:
Using [the recommended `screen`](https://testing-library.com/docs/dom-testing-library/api-queries#screen) cleans this up a bit.

```suggestion
    render(
      <TileCatalogNew
        title="Test Tile Catalog"
        tiles={getTiles(4, 'Tile')}
        numColumns={2}
        numRows={2}
      />
    );

    expect(screen.queryByText('Next page')).toBeNull();
```